src/gribcreate.f

```diff
@@ -1,60 +1,50 @@
+!>    @file
+!>    @brief This subroutines initializes a new GRIB2 message and packs
+!>    GRIB2 Indicator Section and Identification Section.
+!>    @author Stephen Gilbert @date 2000-04-28
+!>
+
+!>    This subroutine initializes a new GRIB2 message and packs GRIB2
+!>    sections 0 (Indicator Section) and 1 (Identification Section).
+!>    This routine is used with routines addlocal(), addgrid(), addfield(),
+!>    and gribend() to create a complete GRIB2 message. Subroutine
+!>    gribcreate must be called first to initialize a new GRIB2 message.
+!>    Also, a call to gribend is required to complete GRIB2 message
+!>    after all fields have been added.
+!>    
+!>    @param[inout] cgrib Character array to contain the GRIB2 message.
+!>    @param[in] lcgrib Maximum Length (in bytes) of array cgrib.
+!>    @param[in] listsec0 Contains information needed for GRIB Indicator
+!>    Section 0. Must be dimensioned >= 2.
+!>    - listsec0(1) Discipline-GRIB Master Table Number (Code Table 0.0)
+!>    - listsec0(2) GRIB Edition Number (currently 2)
+!>    @param[in] listsec1 Contains information needed for GRIB Identification
+!>    Section 1. Must be dimensioned >= 13.
+!>    - listsec1(1) Id of orginating centre (Common Code Table C-1)
+!>    - listsec1(2) Id of orginating sub-centre (local table)
+!>    - listsec1(3) GRIB Master Tables Version Number (Code Table 1.0)
+!>    - listsec1(4) GRIB Local Tables Version Number (Code Table 1.1)
+!>    - listsec1(5) Significance of Reference Time (Code Table 1.2)
+!>    - listsec1(6) Reference Time - Year (4 digits)
+!>    - listsec1(7) Reference Time - Month
+!>    - listsec1(8) Reference Time - Day
+!>    - listsec1(9) Reference Time - Hour
+!>    - listsec1(10) Reference Time - Minute
+!>    - listsec1(11) Reference Time - Second
+!>    - listsec1(12) Production status of data (Code Table 1.3)
+!>    - listsec1(13) Type of processed data (Code Table 1.4)
+!>    @param[out] ierr Error return code.
+!>    - 0 no error
+!>    - 1 Tried to use for version other than GRIB Edition 2
+!>
+!>    @note This routine is intended for use with routines
+!>    addlocal(), addgrid(), addfield(), and gribend() to create a
```

Comment:
@Hang-Lei-NOAA I see doxygen links to addlocal() and gridend(). But not to addgrid() and addfield(). Does anyone know why?

Reply:
It was the warnings from other two code. I will get it fixed in next commit.
Please see this, it has been fixed.
https://hang-lei-noaa.github.io/NCEPLIBS/NCEPLIBS-g2/html/gribcreate_8f.html